python/lsst/ts/xml/data/sal_interfaces/MTAOS/MTAOS_Telemetry.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsl"?>
+<SALTelemetrySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsd">
+  <SALTelemetry>
+    <Subsystem>MTAOS</Subsystem>
+    <EFDB_Topic>MTAOS_m1m3BendingModes</EFDB_Topic>
+    <Description>Measured M1M3 bending modes values.</Description>
+    <item>
+      <EFDB_Name>value</EFDB_Name>
+      <Description>Amplitude of the measured bending mode.</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>micron</Units>
+      <Count>20</Count>
+    </item>
+  </SALTelemetry>
+  <SALTelemetry>
+    <Subsystem>MTAOS</Subsystem>
+    <EFDB_Topic>MTAOS_m2BendingModes</EFDB_Topic>
+    <Description>Measured M2 bending modes values.</Description>
+    <item>
+      <EFDB_Name>value</EFDB_Name>
+      <Description>Amplitude of the measured bending mode.</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>micron</Units>
+      <Count>20</Count>
+    </item>
```

Comment:
How do we realized this in code? The unit of M1M3 and M2 is the Newton for each actuator, not the bending mode in um. We could estimate I guess, but not measure it directly for sure.